thoth/storages/data/alembic/versions/c1839dfcdf13_add_importpackage_foundimportpackage_.py

```diff
@@ -0,0 +1,47 @@
+"""Add ImportPackage, FoundImportPackage classes and get_python_package_version_import_packages_all function
+
+Revision ID: c1839dfcdf13
+Revises: 65ae36e5c38a
+Create Date: 2021-08-12 18:34:58.972606+00:00
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = 'c1839dfcdf13'
+down_revision = '65ae36e5c38a'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.create_table('import_package',
+    sa.Column('id', sa.Integer(), autoincrement=True, nullable=False),
+    sa.Column('import_package_name', sa.Text(), nullable=True),
+    sa.PrimaryKeyConstraint('id')
+    )
+    op.create_table('found_import_package',
+    sa.Column('python_package_version_id', sa.Integer(), nullable=False),
+    sa.Column('import_package_id', sa.Integer(), nullable=False),
+    sa.ForeignKeyConstraint(['import_package_id'], ['import_package.id'], ondelete='CASCADE'),
+    sa.ForeignKeyConstraint(['python_package_version_id'], ['python_package_version.id'], ondelete='CASCADE'),
+    sa.PrimaryKeyConstraint('python_package_version_id', 'import_package_id')
+    )
+    # ### end Alembic commands ###
+
+
+def downgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.drop_constraint(None, 'python_package_version_entity_rule', type_='unique')
+    op.alter_column('has_symbol', 'versioned_symbol_id',
```

Comment:
Also line 39 to 44 should be manually removed

Reply:
was removed